### PR TITLE
Added functionanlity to push tables using false value.

### PR DESCRIPTION
**Pull request by GuidoS**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -6,9 +6,16 @@ var debug = require('debug')('fromPostgres');
...
+  var nogeom = false;
+
+  if (geom === false){
+      nogeom = true;
+  }
+
   geom = geom || 'shape';
   primary = primary || 'objectid';
   var db = knex({
+      debug: true,
```
Comment (calvinmetcalf) [inline index.js]: this shouldn't be left in

File: index.js [hunk 2]
```diff
@@ -26,19 +33,21 @@ function fromPostgres(connection, table, geom, primary, offset, limit) {
...
   }
   var out = new Transform({
       objectMode: true,
       transform: function (chunk, _, next) {
         var geo;
-        try {
-          geo = JSON.parse(chunk.__geom__);
-        } catch (e) {
-          return next(e);
+        if (chunk.__geom__npm ) {
```
Comment (calvinmetcalf) [inline index.js]: I think this is a typo

File: index.js [hunk 3]
```diff
@@ -6,9 +6,16 @@ var debug = require('debug')('fromPostgres');
 module.exports = fromPostgres;
 
 function fromPostgres(connection, table, geom, primary, offset, limit) {
+  var nogeom = false;
```
Comment (calvinmetcalf) [inline index.js]: we could probably rewrite this as `var nogeom = (geom === false);`

**Comment by GuidoS** [comment]
Added updates based on comment to nogeom repo

**Pull request merged**